code/modules/clothing/head/misc.dm

```diff
@@ -280,5 +280,5 @@
 
 /obj/item/clothing/head/crown/fancy
 	name = "magnificent crown"
-	desc = "A crown worn by only the highest emperors of the land."
+	desc = "A crown worn by only the highest emperors of the <s>land</s> space."

```

Comment:
The joke is that using land in that place is proper grammar, but it's being juxtaposed by substituting in space instead.